lib/ic/sensor/adt7420.yaml

```diff
@@ -0,0 +1,23 @@
+datasheet: https://www.analog.com/media/en/technical-documentation/data-sheets/ADT7420.pdf
+description: 16-Bit Digital Temperature Sensor
+footprint: agg:LFCSP-16-AD
+name: ADT7420
+ordercodes:
+- [Farnell, 2306581]
```

Comment:
Order codes should be quoted so they are interpreted as strings instead of integers